### PR TITLE
fix: Styled input field

### LINKED ISSUE #473: Bug: Icons in the input field of the hero section could be aligned
### Describe the bug

Icons in the input field of the hero section in the [page](https://hot.opensauced.pizza/) could be aligned with a simple flex;

### Steps to reproduce

1. Go to https://hot.opensauced.pizza/ hero section of the page to see the input field.

### Browsers

Chrome

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Sadeedpv**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Icons in the input field of the hero section in the [page](https://hot.opensauced.pizza/) could be aligned with a simple flex and modifying tailwind-css at line 76 and adding an extra div at line 77;

## Related Tickets & Documents

Fixes #473 

## Mobile & Desktop Screenshots/Recordings

Before:

before

After:

PR

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
@Sadeedpv can you share a before and after?

File: src/components/Hero.tsx [hunk 1]
```diff
@@ -73,35 +73,37 @@ const Hero = () => {
...
-                setTimeout(() => {
-                  setFocus(false);
-                }, 200)}
-              onKeyUp={(event: React.KeyboardEvent) => {
-                if (event.key === "Enter") {
-                  window.open(comboBoxSelection, "_blank", "noreferrer");
-                }
-              }}
+        <div className="mt-11 px-4 gap-x-2.5 py-2.5  bg-white shadow-2xl rounded-2xl md:min-w-[26.375rem] flex justify-between">
+          <div className="flex">
```
Comment (bdougie) [inline src/components/Hero.tsx:77]: Can you clarify this is the only fix and add that context to your PR description?

Comment (Sadeedpv) [reply src/components/Hero.tsx:77]: Done 👍

**Comment by Sadeedpv** [comment]
Please merge this PR

File: src/components/Hero.tsx [hunk 2]
```diff
@@ -73,35 +73,37 @@ const Hero = () => {
...
-              onBlur={() =>
-                setTimeout(() => {
-                  setFocus(false);
-                }, 200)}
-              onKeyUp={(event: React.KeyboardEvent) => {
-                if (event.key === "Enter") {
-                  window.open(comboBoxSelection, "_blank", "noreferrer");
-                }
-              }}
+        <div className="mt-11 px-4 gap-x-2.5 py-2.5  bg-white shadow-2xl rounded-2xl md:min-w-[26.375rem] flex justify-between">
```
Comment (takanome-dev) [inline src/components/Hero.tsx:76]:
```suggestion
        <div className="mt-11 px-4 py-2.5  bg-white shadow-2xl rounded-2xl md:min-w-[26.375rem] flex justify-between">
```

File: src/components/Hero.tsx (hunk 1 above)
Comment (takanome-dev) [inline src/components/Hero.tsx:77]:
```suggestion
          <div className="flex items-center gap-x-2.5">
```

**Review by takanome-dev — COMMENTED** [review]
Looks good :+1: Just some tweaks and this should be good to go

The `searchNormal` icon is bigger and there is not enough space between the icon and the search input

**Review by NsdHSO — COMMENTED** [review]
LGTM, Sorry for being out of context, can you add [here](https://github.com/open-sauced/hot/blob/beta/src/components/Hero.tsx#L115) `fetchedData?.length`

**Review by takanome-dev — COMMENTED** [review]
LGTM

**Review by bdougie — APPROVED** [review]

**Pull request merged**